### PR TITLE
chore(webtransport-websys): Change logs to debug level

### PR DESCRIPTION
**Pull request by oblique**
## Description

When you enable Kademlia in a large network that uses multiple type of transports, then this log creates a very noisy console and makes meaningful logs hard to see.

We can also remove the log entirely if you prefer. Check: https://github.com/libp2p/rust-libp2p/pull/4015#discussion_r1229951440, https://github.com/libp2p/rust-libp2p/issues/4072, https://github.com/libp2p/rust-libp2p/pull/4133

## Change checklist

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] A changelog entry has been made in the appropriate crates

### CODE REVIEW

File: transports/webtransport-websys/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.2.1
+
+* Increase logs to debug level.
```
Comment (zvolin) [inline transports/webtransport-websys/CHANGELOG.md]:
```suggestion
* Decrease logs to debug level.
```

**Review by zvolin — APPROVED** [review]
can confirm, it's really noisy

File: transports/webtransport-websys/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,11 @@
-## 0.2.1 
+<<<<<<< HEAD
```
Comment (dariusc93) [inline transports/webtransport-websys/CHANGELOG.md]: Looks like a bad merge?

**Review by dariusc93 — CHANGES_REQUESTED** [review]

File: transports/webtransport-websys/CHANGELOG.md (hunk 2 above)
Comment (oblique) [reply transports/webtransport-websys/CHANGELOG.md]: oops. sorry. fixed

**Comment by oblique** [comment]
For some reason CI got stuck

**Review by jxs — COMMENTED** [review]
Thanks for this. Interesting `cargo-semver-checks` does not complain, in fact I'd be more comfortable releasing this as a new minor wdyt?

**Comment by oblique** [comment]
Do you mean 0.3.0? I'm ok with it.

**Comment by jxs** [comment]
> Do you mean 0.3.0? I'm ok with it.

yeah exactly, ok thanks

**Comment by oblique** [comment]
Question: In that case we will need to wait for libp2p 0.54 to incorporate this and #5390 in our crate. Do you have an approximate date for the release? Do you work with release cycles?

**Comment by jxs** [comment]
> Question: In that case we will need to wait for libp2p 0.54 to incorporate this and #5390 in our crate. Do you have an approximate date for the release? Do you work with release cycles?

we're waiting on https://github.com/libp2p/rust-libp2p/issues/5389 to release a new `libp2p` version, but meanwhile I can release `libp2p-webtransport-websys` and I think you can then depend on it directly, wdyt?

**Comment by oblique** [comment]
Sounds good!

**Review by jxs — APPROVED** [review]
LGTM

**Comment by jxs** [comment]
@oblique released `libp2p-webtransport-sys`  [0.3.0](https://crates.io/crates/libp2p-webtransport-websys)

**Pull request merged**